ephios/templates/base.html

```diff
@@ -54,6 +54,7 @@
         <script type="text/javascript" src="{% static "bootstrap/js/bootstrap.bundle.min.js" %}"></script>
         <script type="text/javascript" src="{% static "select2/js/select2.js" %}"></script>
         <script type="text/javascript" src="{% static "django_select2/django_select2.js" %}"></script>
+        <script type="text/javascript" src="{{ select2_translation_url }}"></script>
```

Comment:
```suggestion
        <script type="text/javascript" src="{% static ""select2/js/i18n/"|add:LANGUAGE_CODE|add:".js" %}"></script>
```

Reply:
can you try this? If it works you don't need the `select2_translation_url` I think.